recipes/workshop-demo/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "workshop-demo" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/workshop-demo-{{ version }}.tar.gz
+  sha256: 46b1825fe450cd8070bb41ba28c6c62f92710fd4632d6929c51f61f3d331dce2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - dask-core
+    - imagecodecs
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - npe2
```

Comment:
Hm, how come we need both `npe2` and `napari-plugin-engine`? I thought the former was the successor of the latter?

Reply:
Cause we might be using both styles for this example? 

@tlambert03 can confirm :)

Reply:
yeah, part of the transition strategy is to continue supporting the first generation engine while people adopt the second.  So there will be a period of time where both are around.  PRs like https://github.com/napari/napari/pull/4015 will make it so that npe2 is the only interface that napari needs to care about (even to support first gen plugins), but we're not quite there yet

Reply:
and yes, I think this example was demonstrated both styles... can't remember though

Reply:
That's correct, this was a demo plugin written to showcase the transition from `napari-plugin-engine` to `npe2`.

Reply:
Perfect, thanks for the clarification!